### DISCUSSION TITLE
"Open C# Project" option in menu.

### DISCUSSION THREAD

**Discussion by plyoung** [godotengine/godot-proposals#8765] [Editor]
It would be nice if there were an option in the main menu, perhaps under `Project > Tools`, or `Project > Tools > C#` to quickly open the C# project with the selected external IDE.

**Comment by DevMimas** [comment]
What do you mean exactly?
The external IDE opens when you click on the .cs file. I use Visual Studio 2022 and it works fine.

But that would be cool. If the external IDE opens simultaneously or sequentially with the opening of the project.

**Reply by plyoung** [reply]
Ye, but I do not want to be scrolling down the File System dock to find a cs script to open VS. A menu entry, similar to what Unity has, to quickly open project with your selected IDE would be nice. Personally I simply open the code project via VS' own launcher these days.
